sqlserver/assets/configuration/spec.yaml

```diff
@@ -713,6 +713,28 @@ files:
         type: number
         example: 1800
         display_default: false
+    - name: schemas_collection 
+      description: |
+        Configure collection of schemas.
+      options:
+        - name: enabled
+          description: |
+            Enable schema collection. Requires `dbm: true`. Defaults to true.
```

Comment:
This should default `false` IMO. This is an experimental feature & we should only enable by default once we have a lot of confidence in it. There were plenty of bugs we had to fix in the postgres schema collection feature since it was created.

Reply:
thx, will change